### DISCUSSION TITLE
Why is the shortcut for copy line Up/Down not working?

### DISCUSSION THREAD

**Discussion by kdb13** [coder/code-server#6322] [Q&A]
I'm using the code-server inside the Coder platform. When I try to copy a line using the shortcut `Ctrl+Shift+Alt+Up/Down` it is not working.

Instead, it selects the words after/before the cursor.

**Comment by lg-nineoneninethreenine** [comment]
Some keybindings are not accepted into code-server by default. It is most probably bound to the system so you have to change the keybinding both ways - on the system, and code-server.

Hope this helps!

**Comment by code-asher** [comment]
For what it is worth, those keybindings work for me (it copies the 
current line and pastes it above or below).  I am using 
code-server v4.14.1 over Firefox on Linux.  Definitely check the 
keybindings page to see if they are correctly bound ("Copy Line 
Up" and "Copy Line Down").

**Reply by kdb13** [reply]
I am using code-server on ChromeOS.

**Reply by code-asher** [reply]
Find anything in the keybindings?  Maybe the defaults are different for ChromeOS for some reason.
